### PR TITLE
Handle invalid data.

### PR DESCRIPTION
**Pull request by vitillo**
See https://bugzilla.mozilla.org/show_bug.cgi?id=1271961.

### CODE REVIEW

File: mozaggregator/aggregator.py [hunk 1]
```diff
@@ -48,27 +48,27 @@ def _map_build_id_key_to_submission_date_key(aggregate):
 
 
 def _sample_clients(ping):
-    sample_id = ping.get("meta", {}).get("sampleId", None)
+    try:
+        sample_id = ping.get("meta", {}).get("sampleId", None)
 
-    if not isinstance(sample_id, (int, float, long)):
```
Comment (mreid-moz) [inline mozaggregator/aggregator.py]: It seems like a good idea to continue verifying that `sample_id` is a numeric value for the "percentage" check later on.

File: mozaggregator/aggregator.py [hunk 2]
```diff
@@ -239,9 +239,13 @@ def _map_ping_to_dimensions(ping):
         os = ping["environment"]["system"]["os"]["name"]
         os_version = ping["environment"]["system"]["os"]["version"]
         e10s = ping["environment"]["settings"]["e10sEnabled"]
+
         if os == "Linux":
             os_version = str(os_version)[:3]
 
+        if not build_id.isnumeric():
```
Comment (mreid-moz) [inline mozaggregator/aggregator.py]: The `isnumeric` function allows weird unicode characters like ⅕ . Pretty unlikely, but we'll hit it eventually ;) A more robust way to check "all digits" would be:
```python
try:
  int(build_id)
except:
  return None
```

**Comment by mreid-moz** [comment]
Looks good, r+

**Comment by vitillo** [comment]
The upload process keeps failing on my machine: 

```
Creating tar archive
removing 'python_mozaggregator-0.2.5.2' (and everything under it)
running upload
Submitting dist/python_mozaggregator-0.2.5.2.tar.gz to https://pypi.python.org/pypi
Upload failed (500): Internal Server Error
error: Upload failed (500): Internal Server Error
```

@mreid-moz could you try please?

**Pull request merged**